migration/idempotent/001-base.sql

```diff
@@ -279,6 +279,30 @@ BEGIN
         RETURN NEW;
    END IF;
 
+   --autovacuum notes: we need to vacuum here for the visibility map optimization
+   --but, we don't want to freeze anything since these tables will be truncated upon
+   --compression anyway (so keep freeze settings to the default).
+   --Make the vacuum less aggressive here by upping the threshold and scale factors. 
+   IF current_setting('server_version_num')::integer >= 130000 THEN
+            EXECUTE format('CREATE TABLE %I.%I(time TIMESTAMPTZ NOT NULL, value DOUBLE PRECISION NOT NULL, series_id BIGINT NOT NULL) WITH 
+                            ( 
+                              autovacuum_vacuum_threshold = 50000, autovacuum_vacuum_insert_threshold=50000, 
+                              autovacuum_vacuum_scale_factor=2.0, autovacuum_vacuum_insert_scale_factor=2.0,
+                              autovacuum_analyze_threshold = 50000,
+                              autovacuum_analyze_scale_factor = 1.0
```

Comment:
This seems rather high. Analyze should generally be very cheap to perform and has positive impact on the performance, so 0.1 or 0.2 should be safe.

Reply:
we've seen some issues with high writes to the stat directory so let's compromise on 0.5